src/TestProblem/Hydro/CCSN/Flag_CCSN.cpp

```diff
@@ -125,14 +123,9 @@ bool Flag_Lightbulb( const int i, const int j, const int k, const int lv, const
    if      ( R * UNIT_L < 3e6 )
       Flag = true;
 
-// (2) density is larger than the threshold in Input__Flag_User
-   else if ( Rho[k][j][i] < Threshold[0] )
-      Flag = false;
-
-// (3) check if the minimum angular resolution (in degree) is reached
-   if ( CCSN_Min_AngRes > 0.0  &&  !Flag )
-      if ( R * CCSN_Min_AngRes < dh * RAD2DEG )
-         Flag = true;
+// (2) check if the minimum angular resolution is reached
+   if ( CCSN_AngRes_Min > 0.0  &&  R * CCSN_AngRes_Min < dh  &&  !Flag )
```

Comment:
move the condition `!Flag` to the first one.

Reply:
I move the `!Flag` to the first place.